blueprints/foundational/config.ts

```diff
@@ -0,0 +1,293 @@
+import {
+  baseConfig,
+  Config,
+  GLOBAL_REGION,
+  Template,
+} from '@superluminar-io/aws-luminarlz-cli/lib/config';
+
+/**
+ * The deployed version of the AWS Landing zone accelerator.
+ */
+export const AWS_ACCELERATOR_VERSION = '1.12.1';
+
+/**
+ * The AWS management account, organization and root organizational unit used in this project.
+ */
+export const MANAGEMENT_ACCOUNT_ID = '<<AWS_MANAGEMENT_ACCOUNT_ID>>';
+export const ORGANIZATION_ID = '<<AWS_ORGANIZATION_ID>>';
+export const ROOT_OU_ID = '<<AWS_ROOT_OU_ID>>';
+
+/**
+ * The email address used for the AWS account root emails.
+ */
+export const AWS_ROOT_EMAIL = '<<AWS_ROOT_EMAIL>>';
+/**
+ * The email address used for the AWS alternate contact emails.
+ * TODO: Optionally, we can uncomment this to use the alternate contact email as alternate contact.
+ */
+// export const AWS_ALTERNATE_CONTACT_EMAIL = '<AWS_ALTERNATE_CONTACT_EMAIL>';
+
+/**
+ * The GitHub Actions organization, repository, and branch to which access is granted to deploy the accelerator-config zip file.
+ * TODO: Optionally, we can uncomment this to grant GitHub Actions access to update the accelerator-config zip file and to upload CDK assets.
+ */
+// export const GITHUB_ORGANIZATION = '<GITHUB_ORGANIZATION>';
+// export const GITHUB_REPOSITORY = 'aws-landing-zone';
+// export const GITHUB_REPOSITORY_MAIN_BRANCH = 'main';
+
+/**
+ * The AWS regions in use. The global region is always us-east-1.
+ */
+export const HOME_REGION = 'eu-central-1';
+export const ENABLED_REGIONS = [GLOBAL_REGION, HOME_REGION];
+
+/**
+ * The groups should be aligned with the groups in the AWS IAM Identity Center.
+ * The group names are also being used for the Owner Cost Category and as possible values for the Owner tag.
+ * TODO: The group values should be replaced with IAM Identity Center group names that actually exist in your project!
+ */
+export const Groups = {
+  awsAdministrator: 'aws-administrator',
+  awsDeveloper: 'aws-developer',
+  awsBillingReviewer: 'aws-billing',
+};
+export type Group = (typeof Groups)[keyof typeof Groups];
+
+/**
+ * The organizational units are deployed and managed via the LZA.
+ * They are also used for the OrganizationalUnit Cost Category.
+ */
+interface OrganizationalUnit {
+  name: string;
+  ignore?: boolean;
+}
```

Comment:
I still find it weird that we declare those here, but on the other hand, they are only used in the config and never in the framework. Maybe in some next iteration it could be part of the cli as some kind of quality of life simplification feature. For now they should stay here, I guess 🤷

Reply:
Yeah I'm also not sure yet what's best here but that's why I would keep them in here for now.

Reply:
exaclty, as said, when we learned some things, we can optimize and improve :)